tests/libmcs/configure.sh

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env bash
+set -e; set -o pipefail
+
+SCRIPT_DIR="$(cd "$(dirname "$0")" && pwd)"
+
+(cd "${SCRIPT_DIR}/repo" && ./configure \
+     --cross-compile="" \
+     --compilation-flags="" \
```

Comment:
What's the default here? Why do we want something different?

Reply:
These were the flags hayroll was using, so I just copied it, since we want to test the same thing.

Reply:
Maybe add a comment explaining where the flags came from? Otherwise looks good.

Reply:
Added in c0ce6aa.